benchmarks/exclusive_display/README.md

```diff
@@ -0,0 +1,87 @@
+# Exclusive Display Benchmark
+
+This document investigates the performance of the [exclusive
+display](https://docs.nvidia.com/holoscan/sdk-user-guide/enable_exclusive_display.html) feature in
+`holoviz` operator of Holoscan SDK.
+
+## Introduction
+
+By default, Holoscan SDK application uses the `holoviz` operator for display, and it uses a
+windowing system like [X11](https://en.wikipedia.org/wiki/X_Window_System). X11 is a compositor
+which combines the display requests from different applications and renders them on the screen. In
+`exclusive_display` mode, `holoviz` turns off the default display manager and directly renders the
+output in full-screen mode. This mode is obviously more performant as a single application
+exclusively uses the monitor to display its output.
+
+In this document, we provide the performance measurements of the `exclusive_display` mode and compare it
+with the default mode that uses `X11`. We execute the [endoscopy tool tracking](https://github.com/nvidia-holoscan/holohub/tree/main/applications/endoscopy_tool_tracking) in these two display
+modes and measure its maximum and average end-to-end latency. In addition, we also run a number of
+"headless" applications simultaneously. These headless applications are executing both AI workloads
+and graphics processing but do not utilize the screen to display any output. They are representative
+of workloads that run alongside a primary display application which is the endoscopy
+tool tracking application using display in this experiment.
+
+## Platform
+
+The experiments are conducted with Holoscan v2.1 container on IGX Orin with RTX A6000 GPU flashed
+with IGX SW 1.0.
+
+## Results
+
+For the experiment, we use the endoscopy tool tracking application which is using the display
+monitor for outputs in two modes, as said above. This application is executed with `realtime: false`
+for the
+video stream replayer source, so that the source feeds the frames as fast as possible, without an
+external frame-rate limitation.
+
+For the headless applications, we run the same endoscopy
```

Comment:
Introduction is about `exclusive` mode, but here it's talking about `headless`.

Reply:
@AndreasHeumann The primary application whose performance is benchmarked is either in "exclusive" mode or "default" mode. The graphs show the performance of this single primary application.
We run a varying number of "headless" applications in the background. The number of these background "headless" applications vary from 0 to 11.
This is clarified in the second paragraph of the Introduction section above.